### PR TITLE
Regenerate using kubebuilder/v3

### PR DESCRIPTION
**Pull request by zchee**
#### Summary

Regenerate using kubebuilder/v3.

#### Ticket Link

N/A

#### Fixes

N/A

### CODE REVIEW

**Comment by zchee** [comment]
@cpanato @dlorenc Initial pull request of development rekor operator. Could you review this? (or, any chance to join this repository maintainer?)

File: main.go [hunk 1]
```diff
@@ -103,26 +99,35 @@ func main() {
 		Port:                    webhookPort,
 		CertDir:                 webhookCertDir,
 		HealthProbeBindAddress:  healthAddr,
-		EventBroadcaster:        broadcaster,
```
Comment (zchee) [inline main.go:106]: `EventBroadcaster` has been deprecated.

File: main.go [hunk 2]
```diff
@@ -144,14 +149,21 @@ func main() {
...
+		&probeAddr,
+		"health-probe-bind-address",
+		":8081",
+		"The address the probe endpoint binds to.",
+	)
+
 	fs.BoolVar(
 		&enableLeaderElection,
-		"enable-leader-election",
+		"leader-elect",
```
Comment (zchee) [inline main.go:166]: kubebuilder changed flag name.

**Comment by cpanato** [comment]
sorry for the delay, i was in vacation

**Pull request merged**